narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -38,14 +38,16 @@ function check_ignore_zip_file {
 
 	local file_url="${zip_directory_url}/${hotfix_zip_file}"
 
-	if [[ "x${IGNORE_ZIP_FILES}" =~ x*${hotfix_zip_file}* ]]
+	if echo "${IGNORE_ZIP_FILES}" | grep -q "${hotfix_zip_file}"
```

Comment:
This change (the check that makes a lot of noise) is already merged in a branch. It should  be an individual commit anyway and should not mix with the other changes related to the ignore list